docs/addons/addon-knowledge-base.mdx

```diff
@@ -71,7 +71,7 @@ Complementing the components, also included is a set of UI primitives. Use the c
 
 <Callout variant="info">
 
-  The color palette implemented by `@storybook/components` is a high-level abstraction of the [`@storybook/theming`](https://github.com/storybookjs/storybook/tree/next/code/lib/theming/src) package.
+  The color palette implemented by `storybook/internal/components` is a high-level abstraction of the [`storybook/theming`](https://github.com/storybookjs/storybook/tree/next/code/lib/theming/src) module.
```

Comment:
When you're able, can you adjust the link here and in the table so that it doesn't throw a 404?